Formula/asdf.rb

```diff
@@ -0,0 +1,37 @@
+class Asdf < Formula
+  desc "Extendable version manager with support for Ruby, Node.js, Erlang & more"
+  homepage "https://github.com/asdf-vm"
+  url "https://github.com/asdf-vm/asdf/archive/v0.3.0.tar.gz"
+  sha256 "1d986789bfcbef76dc23cc968ca192a03862a9496df65e7cd5d6f8117bc85789"
+
+  bottle :unneeded
+
+  depends_on "coreutils" => :run
+  depends_on "automake" => :run
+  depends_on "autoconf" => :run
+  depends_on "openssl" => :run
+  depends_on "libyaml" => :run
+  depends_on "readline" => :run
+  depends_on "libtool" => :run
+  depends_on "unixodbc" => :run
+
+  def install
+    bash_completion.install "completions/asdf.bash"
+    prefix.install Dir["*"]
+  end
+
+  def caveats; <<-EOS.undent
+    #########################################
```

Comment:
Don't add these lines.  Keep it concise, like [`bash-preexec.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bash-preexec.rb).